ClassificationOfSubgroups/Ch5_PropertiesOfSLOverAlgClosedField/S4_PropertiesOfCentralizers.lean

```diff
@@ -47,6 +43,13 @@ theorem centralizer_s_eq_SZ {σ : F} (hσ : σ ≠ 0) : centralizer { s σ } = S
     rw [hy]
     simp [add_comm]
 
+theorem centralizer_s_eq_neg_s {σ : F}:
```

Comment:
not sure if this will be useful, but realized that the negative part wasn't formalized